### PR TITLE
third chapter- advance concepts of mershery

### LINKED ISSUE #1987: [Learn] Add a new course "Advance concepts of Service Mesh"
**Current State:**
We have only 1 course `introduction to service mesh` under the learning path `mastering service mesh`, the `advance concepts of service mesh` course is not yet added.

**Desired State:**
A new course `Advance concepts of service mesh` needs to be added under the learning path `mastering service mesh` [here](https://github.com/layer5io/layer5/tree/master/content-learn/mastering-meshery-istio), it should use the content from this [repo](https://github.com/layer5io/advanced-istio-service-mesh-workshop) and have a structure similar to this [course](https://github.com/layer5io/layer5/tree/master/content-learn/mastering-meshery-istio/introduction-to-service-meshes)

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by The-Loony-Coder**
Signed-off-by: Mritunjay <msengar121@gmail.com>

**Third chapter- advance course**

This PR fixes #1987

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 88ef36c2467444dd633ffea00aa7755090aca7bc at: https://60ffb3af5288d592ab1febf8--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Update the commit-msg to use `Updates` instead of `Fixes`

File: content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/index.mdx [hunk 1]
```diff
@@ -31,6 +31,9 @@ meshesYouLearn:
 toc:
   [
     'getting-started',
+    'deploy-an-application',
```
Comment (Nikhil-Ladha) [inline content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/index.mdx:34]: This chapter is not a part of this PR, please remove it.

File: content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/deploy-an-application.mdx [hunk 2]
```diff
@@ -0,0 +1,218 @@
+---
```
Comment (Nikhil-Ladha) [inline content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/deploy-an-application.mdx:1]: Same as above

File: content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/expose-services.mdx [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+To just get the first port of istio-ingressgateway service, we can run this:
+
+```sh
+kubectl get service istio-ingressgateway -n istio-system --template='{{(index .spec.ports 1).nodePort}}'
+```
+
+<h3 class="chapter-sub-heading"> Create a DNS entry:</h3>
+
+Modify you local `/etc/hosts` file to add an entry for your sample application.
```
Comment (Nikhil-Ladha) [inline content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/expose-services.mdx:96]: nit: `you` -> `your`

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 1e7f9c3490f3fc5a8cdb9c8889ea2db924f3631f at: https://610cf029fdc6d37cf1d40a52--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 619311743e7cf053483836869f6427deb4fc6f28 at: https://610cf86f2175638d3e6a014f--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**